app/API/V1/Repositories/RoleRepository.php

```diff
@@ -83,6 +84,30 @@ public function removePermissions(array $rolePermissions, bool $delete=false):vo
         }
     }
 
+    /**
+     * Set user's default role
+     *
+     * @param User $user
+     * @throws \Doctrine\DBAL\ConnectionException
+     */
+    public function setUserPermissions(User $user):void
```

Comment:
See my comments above on small changes to make here

Reply:
This one is changed